creator/Documents/ScoreboardIntroduction.md

```diff
@@ -44,11 +44,11 @@ Because the objectives we are tracking do not already exist in the game, you wil
 
     ![Image of a scoreboard objective successfully displayed in a sidebar](Media/Commands/scoreboard_setdisplay_sidebar.png)
 
-1. Now that our objective exists, add a player named "SomeRandomPlayer" and give them a score of 0. Enter `/scoreboard players add SomeRandomPlayer shearedSheep 0`. The player's name will be displayed on the sidebar under Top Shearers.
+1. Now that our objective exists, add a player named "SomeRandomPlayer" and give them a score of 0. Enter `/scoreboard players set SomeRandomPlayer shearedSheep 0`. The player's name will be displayed on the sidebar under Top Shearers.
```

Comment:
For lines 47 and 51, I felt that `/scoreboard players set` was more appropriate for the operation being performed here. Using `add` on a null value works the same way, but is predicated on the value being reset.